101-servicebus-create-namespace/README.md

```diff
@@ -0,0 +1,7 @@
+# 101-servicebus-create-namespace
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-servicebus-create-namespace%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
```

Comment:
We had this discussion with you and our PM's before and we had decided to not have this button untill we work with the Visualize team and have it working for us.